### ISSUE TITLE
Suggestion! If possible , making it supporting, complying and optional toward multi-cloud (AWS, GCS and Azure) would be a great plus!

### ISSUE THREAD

**Issue by emg110** [WaferFinance/AlgoNode-AWS#1]
Great job! Congrats!
For future steps Instead of single cloud implementation (which has proved poor availability choice these days) why not a multi-cloud approach with support for major cloud vendors (AWS, GCS, and Azure). Even parallel multi-node implementation to gain maximum reliability and availability could come upon future options then. Just a thought!

**Comment by Blackglade** [comment]
Yup that's eventually the goal! Would likely need rewrite all of this and transition it over to something like terraform / create custom images for each cloud provider to allow for multi-cloud deployments.

For now, the initial release was primarily AWS focused. As interest grows in this, will definitely consider expanding the number of options users have. Thanks for the feedback :)
